book/src/SUMMARY.md

```diff
@@ -16,6 +16,8 @@
   - [Mining](user/mining.md)
     - [Testnet Mining with s-nomp](user/mining-testnet-s-nomp.md)
   - [Kibana blockchain explorer](user/elasticsearch.md)
+  - [Improving Perfomance](user/performance.md)
```

Comment:
Nitpick: spelling

```suggestion
  - [Improving Performance](user/performance.md)
```

Reply:
I'm actually going to put this section into "Troubleshooting", so I'm not going to commit this suggestion.